app/models/manageiq/providers/openstack/inventory/collector/target_collection.rb

```diff
@@ -272,6 +274,10 @@ def infer_related_ems_refs!
       infer_related_orchestration_stacks_ems_refs_db!
       infer_related_orchestration_stacks_ems_refs_api!
     end
+    unless references(:cloud_volumes).blank?
+      infer_related_orchestration_stacks_ems_refs_db!
```

Comment:
Shouldn't it be `infer_related_cloud_volumes_ems_refs_db!` ?

Reply:
Right, thanks fot the catch, fixed in https://github.com/ManageIQ/manageiq-providers-openstack/pull/400